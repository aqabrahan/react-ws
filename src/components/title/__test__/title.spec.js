const React = require('react');
const Renderer = require('react-test-renderer/shallow');
const Component = require('../src/TitlePage');

let props = {
  title: 'title',
};

describe('Title Component', () => {
  const renderer = new Renderer();

  it('should render component', () => {
    const component = renderer.render(<Component {...props} />);
    expect(component).toMatchSnapshot();
  });

  it('should render component with subtitle', () => {
    props = {
      ...props,
      subtitle: 'subtitle',
    };
    const component = renderer.render(<Component {...props} />);
    expect(component).toMatchSnapshot();
  });
});
