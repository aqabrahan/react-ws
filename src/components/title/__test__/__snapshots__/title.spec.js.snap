// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component should render component 1`] = `
<div
  className="title-content"
>
  <h2
    className="title-content__main"
  >
    title
  </h2>
  
</div>
`;

exports[`Title Component should render component with subtitle 1`] = `
<div
  className="title-content"
>
  <h2
    className="title-content__main"
  >
    title
  </h2>
  <span
    className="title-content__subtitle"
  >
    subtitle
  </span>
</div>
`;
